cmd/skopeo/utils.go

```diff
@@ -9,12 +9,20 @@ import (
 	"github.com/urfave/cli"
 )
 
-func contextFromGlobalOptions(c *cli.Context, credsFlag string) (*types.SystemContext, error) {
+func contextFromGlobalOptions(c *cli.Context, credsFlag, certPathFlag, tlsVerifyFlag string) (*types.SystemContext, error) {
 	ctx := &types.SystemContext{
-		RegistriesDirPath:           c.GlobalString("registries.d"),
+		RegistriesDirPath: c.GlobalString("registries.d"),
+		// DEPRECATED: keep this here for backward compatibility, but override
+		// them if per subcommand flags are provided (see below).
```

Comment:
This could entirely go away since we changed the way we lookup certs in `--cert-path` (no more `cert.pem`, `cert.key`), @mtrmac what do you think about getting rid of the global flags altogether?